forte/casscf/casscf_orb_grad.cc

```diff
@@ -661,6 +661,30 @@ void CASSCF_ORB_GRAD::format_fock(psi::SharedMatrix Fock, ambit::BlockedTensor F
     });
 }
 
+psi::SharedMatrix CASSCF_ORB_GRAD::fock(std::shared_ptr<RDMs> rdms) {
+    // put spin-summed 1RDM to psi4 Matrix
```

Comment:
A function like this that converts an ambit tensor to a psi4 matrix either already exists or should be implemented as a separate function because this it the kind of code we do not want to have repeated all over forte. Can you please double check and if not present implement somewhere (helpers maybe?).

Reply:
`tensor_to_matrix` is  the one I uesed

Reply:
Yes, I thought that only returns a matrix without symmetry.